packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss

```diff
@@ -5,7 +5,7 @@ cxl-vaadin-accordion {
   /**
    * Theme "cxl-accordion-card".
    */
-  &[theme~="cxl-accordion-card"] {
+  &[theme~="cxl-accordion-card"] { display: grid; grid-gap: var(--lumo-space-m); grid-template-columns: repeat( auto-fill, minmax( calc( var(--cxl-wrap-width) / 4 ), 1fr ) );
```

Comment:
Prettier did not auto-break it into Aybolit's multiple lines formatting :thinking: Can you update manually?